Help Desk process

```diff
@@ -0,0 +1,31 @@
+Process for Help Desk Requests from External Users
+
+When a new issue is received from an external user, it automatically creates a JIRA ticket.
+Help Desk managers send a generic response (within one day) to the user notifying them that we will get them an answer soon
+Unless the Help Desk managers can answer the question themselves, they assign the issue to the relevant box lead.
+The box lead assigns external issues within two days to someone who can address the issue.
```

Comment:
Well, that's up to the box lead. There might be bugs that are critical enough that they might get assigned to sprinters if they can't be fixed quickly by non-sprinters.